src/components/node-list/node-list-row-list.js

```diff
@@ -51,31 +51,22 @@ const NodeRowList = ({
           style={lowerStyle}
         />
         {items.slice(start, end).map((item) => (
-          <NodeListRow
-            container="li"
-            key={item.id}
+          <FilterRow
```

Comment:
same here, why do we need the ul/li code now, since FilterRow is a div. when I inspect the code, it looks a bit confusing?

Reply:
same as above